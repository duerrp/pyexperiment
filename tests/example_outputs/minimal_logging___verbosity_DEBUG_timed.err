[[X;XXmDEBUG[Xm   ] [X.XXXs] [XmStart: './examples/minimal_logging --verbosity DEBUG timed'[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [XmTime: 'XXXX-XX-XX XX:XX:XX.XXXXXX'[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmsetup took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [Xmfor took X.XXXXXXs[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [XmEnd: './examples/minimal_logging --verbosity DEBUG timed'[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [XmTime: 'XXXX-XX-XX XX:XX:XX.XXXXXX'[Xm
[[X;XXmDEBUG[Xm   ] [X.XXXs] [XmTook: X.XXXs[Xm
