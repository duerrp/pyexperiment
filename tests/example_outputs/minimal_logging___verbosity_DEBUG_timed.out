[X;XXmTiming Summary:[Xm
	'setup': X.XXXXXXs
	'for' (timed XX times): total = X.XXXXXXs, min = X.XXXXXXs, max = X.XXXXXXs, median = X.XXXXXXs
